public/app/features/dashboard/utils/tracking.ts

```diff
@@ -0,0 +1,24 @@
+import { reportInteraction } from '@grafana/runtime';
+
+import { DashboardModel } from '../state';
+
+export function trackDashboardLoaded(dashboard: DashboardModel) {
+  // Count the different types of variables
+  const variables = dashboard.templating.list
+    .map((v) => v.type)
+    .reduce((r, k) => {
+      r[variableName(k)] = 1 + r[variableName(k)] || 1;
+      return r;
+    }, {});
+
+  reportInteraction('dashboard_loaded', {
+    uid: dashboard.uid,
+    title: dashboard.title,
+    style: dashboard.style,
+    schemaVersion: dashboard.schemaVersion,
```

Comment:
is this before or after migrations?  if after... it will always be equal to the deployed version

Reply:
> is this before or after migrations? if after... it will always be equal to the deployed version

It is after migrations. I'm not sure what that means that it is same as the deployed version. Would it make sense to do the tracking before migrations?

Reply:
I'd say, it is better to track it after the migrations for consistency, but if we want to know the persisted schema version, we should store it before migrating it, and then send it here.

Reply:
I've added a before migration version as well.